docs/api_guide/plot_06_sklearn_pipeline_cv_demo.py

```diff
@@ -389,6 +433,9 @@ def highlight_max_cell(cvdf_wide, ax):
 ax.legend()
 sns.despine(ax=ax)
 
+# %%
+# The plot confirms that the firing rate distribution is accurately captured by our model predictions.
```

Comment:
given how similar the loglikelihood is for MSplineBasis(10) and RaisedCosineBasisLinear(10), I think it would be interesting to plot their predictions as well.